<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QMainWindow" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LoginWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="LoginBorder" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#LoginBorder
{
	background-color:rgb(255, 255, 255);
}</string>
    </property>
    <widget class="QComboBox" name="accountComboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>169</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>280</y>
       <width>169</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="headPix">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>232</y>
       <width>91</width>
       <height>81</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>240</y>
       <width>77</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="registerAccount">
        <property name="styleSheet">
         <string notr="true">
</string>
        </property>
        <property name="text">
         <string>注册账号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findPassword">
        <property name="text">
         <string>找回密码</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>191</width>
       <height>18</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="savePassword">
        <property name="text">
         <string>记住密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="autoLogin">
        <property name="text">
         <string>自动登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
